pkg/client/integration_test.go

```diff
@@ -217,6 +217,7 @@ func RunAuthorizationCodeFlow(t *testing.T, opServer *httptest.Server, clientID,
 		targetURL,
 		[]string{"openid", "email", "profile", "offline_access"},
 		rp.WithPKCE(cookieHandler),
+		rp.WithAuthStyle(oauth2.AuthStyleInHeader),
```

Comment:
Think you forgot the import,

Reply:
Damn, I always worked with GoLand (does the import for me) and I always forget it, if i'm not using it.